app/controllers/application_controller.rb

```diff
@@ -56,6 +56,6 @@ def current_path(*); end
   def next_path(*); end
 
   def current_app
-    @current_app ||= App.find_or_create_by!(user: current_user)
+    @current_app ||= MbSnapApp.find_or_create_by!(user: current_user)
```

Comment:
Yeah I was going back and forth on what to call this. Get Calfresh Calls this class `CalfreshApplication`. Overall, the word `Application` is annoying. So I prefer "App"...but maybe that is confusing. Also, the challenge with using shorthand is it's easy to forget. For example, in GCF they call things "docs" and "documents" in different places. Very maddening. 

I think `SnapApplication` could work. Based on above, what do you think is best?